main/main.cpp

```diff
@@ -2619,6 +2619,20 @@ bool Main::start() {
 
 	OS::get_singleton()->set_main_loop(main_loop);
 
+	uint64_t minimum_time_msec = GLOBAL_DEF("application/boot_splash/minimum_display_time_msec", 0);
+	ProjectSettings::get_singleton()->set_custom_property_info("application/boot_splash/minimum_display_time_msec",
+			PropertyInfo(Variant::INT,
+					"application/boot_splash/minimum_display_time_msec",
+					PROPERTY_HINT_RANGE,
+					"0,100,1,or_greater")); // No negative numbers
```

Comment:
You can add `suffix:ms` and remove `_msec` from the name I think.

Reply:
Done.

Reply:
Note: We should rename other similar settings.

![](https://user-images.githubusercontent.com/47700418/176225947-7c95878f-28e4-4c88-b433-5c14031f3add.png)

Reply:
Yeah the suffix is a new feature, we could use it consistently for this kind of setting/property.